### DISCUSSION TITLE
Enabling Two Factor after updating has essentially locked me out of my account.

### DISCUSSION THREAD

**Discussion by ghost** [OrchardCMS/OrchardCore#15177] [Q&A]
After checking to box to require 2fa for all accounts,  I was met with this screen on my next sign in.  But I have no way to accept the policy or enable 2fa for the account.  Any help would be greatly appreciated.

![2fa](https://github.com/OrchardCMS/OrchardCore/assets/96081566/544ecc83-7a13-4c89-b205-fe5f54e45c6a)

**Comment by Piedone** [comment]
I'm late to the party, but this happens if you have `ITrackingConsentFeature` of ASP.NET Core enabled (like with our [Privacy module](https://github.com/Lombiq/Orchard-Privacy), but have not accepted its consent request. If you have some cookie popup-blocking browser extension installed, like [I don't care about cookies](https://chromewebstore.google.com/detail/i-dont-care-about-cookies/fihnjjcciajhdojfnbdddfaoknhalnja), then you won't see the request and will get this message.

You need to disable the blocking extension for the site and accept the consent, or disable `ITrackingConsentFeature`.
